containers/src/Kokkos_DynRankView.hpp

```diff
@@ -1194,30 +736,37 @@ class DynRankView : public ViewTraits<DataType, Properties...> {
 
   //----------------------------------------
   // Memory span required to wrap these dimensions.
-  static constexpr size_t required_allocation_size(
-      const size_t arg_N0 = 0, const size_t arg_N1 = 0, const size_t arg_N2 = 0,
-      const size_t arg_N3 = 0, const size_t arg_N4 = 0, const size_t arg_N5 = 0,
-      const size_t arg_N6 = 0, const size_t arg_N7 = 0) {
-    return map_type::memory_span(typename traits::array_layout(
-        arg_N0, arg_N1, arg_N2, arg_N3, arg_N4, arg_N5, arg_N6, arg_N7));
-  }
-
+  /*
+    static constexpr size_t required_allocation_size(
+        const size_t arg_N0 = 0, const size_t arg_N1 = 0, const size_t arg_N2 =
+    0, const size_t arg_N3 = 0, const size_t arg_N4 = 0, const size_t arg_N5 =
+    0, const size_t arg_N6 = 0, const size_t arg_N7 = 0) { return
+    map_type::memory_span(typename traits::array_layout( arg_N0, arg_N1, arg_N2,
+    arg_N3, arg_N4, arg_N5, arg_N6, arg_N7));
+    }
+  */
```

Comment:
Did you mean to remove or keep? Looks like it's not used anymore.
```suggestion
```

Reply:
Yeah great lol, this is yet another untested public function in the old DynRankView interface, we likely still need it ...

Reply:
What's also broken about it is that it takes 8 arguments (like some other things) even though the underlying View is only 7 dimensional??

Reply:
I had commented it out because of the map_type reliance, to fix it later. But since it actually doesn't get tested I never got back to fixing it ...

Reply:
I put it back in.